UPGRADE-15.0.md

```diff
@@ -1037,6 +1037,10 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 
 -   see #project-base-diff to update your project
 
+#### don't add manually added order item into repeat order cart ([#3212](https://github.com/shopsys/shopsys/pull/3212))
+
+-   see #project-base-diff to update your project
+
```

Comment:
The change made in this PR is done in the `frontend-api` package so there is no change in project-base so you do not need UPGRADE notes.

Reply:
ok, i removed that commit then